tests/models/test_dependencies_schema.py

```diff
@@ -209,5 +215,22 @@ def test_multiple_set_retriever_schema_with_same_name():
             other_columns=["column1", "column2"],
         )
 
-    # If there is an error, the schema is cleared
-    assert _get_retriever_schema() == []
+    with _get_dependencies_schemas() as schema:
+        assert schema.to_dict()["dependencies_schemas"] == {
+            DependenciesSchemasType.RETRIEVERS.value: [
+                {
+                    "doc_uri": "doc-uri",
```

Comment:
Related to the comment above, as a user would I expect the second `set_retriever_schema` call to take precedence?

Reply:
Nvm, see that that's already what's happening